python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_l3_edge.schema.yml

```
@@ -20,7 +20,8 @@ $defs:
               description: P2P pool name.
             ipv4_pool:
               type: str
-              description: IPv4 address/Mask.
+              description: Comma separated list of prefixes (IPv4 address/Mask) or ranges (IPv4_address-IPv4_address).
```

```suggestion
              description: Comma-separated list of prefixes (IPv4 address/prefix length) or ranges (IPv4_address-IPv4_address).
```
Same in other places as well

---

I was just using what we had before. I will keep the /Mask for now and we can go a full round everywhere to align.